src/runtime_src/core/pcie/driver/linux/xocl/userpf/xocl_errors.c

```diff
@@ -0,0 +1,62 @@
+/*
+ * A GEM style device manager for PCIe based OpenCL accelerators.
+ *
+ * Copyright (C) 2016-2021 Xilinx, Inc. All rights reserved.
```

Comment:
this is a new file, so the Copyright year should be 2021 not 2016-2021

Reply:
done